pom.xml

```diff
@@ -2574,6 +2579,10 @@
                                         <groupId>org.alluxio</groupId>
                                         <artifactId>alluxio-core-transport</artifactId>
                                     </dependency>
+                                    <dependency>
```

Comment:
What are the conflicts? Can we fix the Alluxio dependencies to not have duplicate classes or resources?